src/features/content-explorer/content-explorer/ContentExplorer.scss

```diff
@@ -72,22 +72,56 @@
         }
     }
 
+    .bdl-ContentExplorer-subheader {
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+
+        .checkbox-container,
+        .toggle-container {
+            margin: 0;
+        }
+
+        .bdl-ContentExplorerSelectAll-checkbox {
+            margin-right: 4.45 * $bdl-grid-unit;
+        }
+    }
+
     .content-explorer-select-all-container {
-        display: block;
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+        width: 100%;
```

Comment:
somewhat related to this comment: https://github.com/box/box-ui-elements/pull/3242/files#r1113512295

do you need these styles? I think `margin-left: auto` on `content-explorer-select-all-checkbox-label` might be enough

Reply:
Yes, I need the width: 100% or else it does not look correct